server/routerlicious/.changeset/dirty-keys-follow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/server-services-shared": minor
+---
+
+Created a utility function for Redis connection handling
+
+Exported a new function that helps shut down Redis connections using the `quit()` command.
```

Comment:
Calling out the function name here would be nice. I know it's not exactly how things work in practice for server, but these changesets should be thought of as the contents of release notes and thus we should try to write them with consumers of the packages in mind (in this case I would think of someone writing their own server implementation on top of the OSS packages). And if that's the target audience, I think it makes a lot of sense for the release notes to say "There's a new function `functionName()` that does X..." so they know what they would have to call to use the new feature.